Demo/build.gradle

```diff
@@ -44,4 +44,17 @@ dependencies {
     testImplementation 'junit:junit:4.+'
     androidTestImplementation 'androidx.test.ext:junit:1.1.3'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.4.0'
+    detektPlugins "io.gitlab.arturbosch.detekt:detekt-formatting:1.18.0-RC2"
+}
+
+detekt {
```

Comment:
Any strong opinions on if we should enable detekt for the Kotlin demo app?

Reply:
i dont mind adding it